website/docs/d/catalog_media.html.markdown

```diff
@@ -47,6 +47,7 @@ The following arguments are supported:
 * `catalog_id` - (Optional; *v3.8.2+*) The ID of the catalog to which the media file belongs. It's mandatory if `catalog` field is not used.
 * `name` - (Required) Media name in catalog (optional when `filter` is used)
 * `filter` - (Optional; *2.9+*) Retrieves the data source using one or more filter parameters
+* `download_to_file` - (Optional; *3.11+*) Will download the contents of the media item into the given file
```

Comment:
What are the limitations in size? What about adding a note that there are limitations?

Reply:
Thanks for the note!